data/text_data.py

```diff
@@ -4855,5 +4855,108 @@
     "Jim Zoet",
     "Bill Zopf",
     "Ivica Zubac",
-    "Matt Zunic"
+    "Matt Zunic",
+    #NBA 2020 Draft Class
```

Comment:
@Nahoyhp this is awesome, thanks for adding all the missing players. Before we merge this however, can you possibly reorder in the names in alphabetical order by last name?